### PR TITLE
Fix #10, #11 - add UMD bundle and transpile using babel7

### LINKED ISSUE #10: Add UMD bundle
By adding a UMD bundle we can support AMD, CommonJS, and old school global objects in a single bundle

### PR DESCRIPTION
**Pull request by jordanaustin**
Transpile the new UMD bundle using babel to help support environments without modern syntax like async/await

### CODE REVIEW

**Comment by jordanaustin** [comment]
This PR also removes the commonjs module in favor of the umd version.

**Comment by spectranaut** [comment]
hey @jordanaustin  -- I'm getting an error while running the scratch-vm unit tests when trying to use the udm module, you can see it here: https://gist.github.com/spectranaut/7d1329548337d9a28c498848e2cbadfc

Because it's easy for me to test this bug locally right now, I'm going to try to see if I can get the regeneratorRuntime included in the module via the rollup config.

**Comment by jordanaustin** [comment]
@spectranaut ah, okay I need to use babel-plugin-transform-runtime then. Thanks for testing it, I'll get a update up in a minute or two

File: rollup.config.js [hunk 1]
```diff
@@ -7,32 +19,22 @@ export default [
...
-      return undefined;
-    }
-  })]
+  plugins: [
+    terserPlugin,
+    babel({
+      babelrc: false,
+      presets: ['@babel/env'],
+      exclude: 'node_modules/**'
+    })
```
Comment (spectranaut) [inline rollup.config.js]: Hi @jordanaustin, I had to really read into babel/rollup config options to build a bundle with the right transformations for node in our case. It turns out this is what we need:
```
    babel({
      presets: [
        ["@babel/env", {
	  "targets": {
	    "node": "8.0.0"
	  },
          "modules": false,
          "useBuiltIns": "usage"
        }]
      ],
      exclude: 'node_modules/**',
    })

```

This also requires a new dependency in package.json -- specifically a dependency not dev dependency, we need it to be installed when requiring @vernier/godirect:
```
  "dependencies": {
    "core-js": "^2.6.1"
  }
```

**Comment by spectranaut** [comment]
ok! I left a comment with the changes to the rollup config to get the regeneratorRuntime included :)

File: rollup.config.js (hunk 1 above)
Comment (jordanaustin) [reply rollup.config.js]: That's annoying, Node 8 has support for async/await - I wonder why Scratch3 doesn't allow async/await

File: package.json [hunk 2]
```diff
@@ -30,5 +33,8 @@
     "webbluetooth"
   ],
   "author": "Vernier Software & Technology",
-  "homepage": "https://github.com/VernierST/godirect-js"
+  "homepage": "https://github.com/VernierST/godirect-js",
+  "dependencies": {
+    "core-js": "^2.6.1"
+  }
```
Comment (spectranaut) [inline package.json]: hey I was just reviewing this again today and retested -- we don't need to install core-js as a dependency, this is was to solve and intermediate problem before I specified node 8.0.0 as the target :)

Comment (jordanaustin) [reply package.json]: @spectranaut that's great! I just updated this PR to remove that dependency

**Pull request merged**